src/js/VirtualFC.js

```diff
@@ -7,7 +7,7 @@ const VirtualFC = {
 
         virtualFC.resetState();
 
-        virtualFC.CONFIG.flightControllerVersion = "4.2.4";
+        virtualFC.CONFIG.flightControllerVersion = "4.3.0";
```

Comment:
This change has nothing to do with presets?

Reply:
Yeah I guess so. Should I remove,l? Figured we should change it to 4.3 anyways.